types/video-react/index.d.ts

```diff
@@ -0,0 +1,510 @@
+         */
+        autoPlay?: boolean;
+
+        /**
+         * Seek the Video at A Specific Time On Load
+         */
+        startTime?: number;
+    }
+
+    interface ShortcutItem {
+        /**
+         * The key code for the shortcut.
+         */
+        keyCode: number;
+        /**
+         * Defines if the action fires when the CTRL
+         * key is pressed.
+         */
+        ctrl: boolean;
+        /**
+         * The function to control the player when
+         * the shortcut is pressed.
+         */
+        handle: (state: PlayerState, actions: PlayerActions) => void;
+    }
+
+    interface ShortcutProps {
+        /**
+         * Allow click to play/pause, default: `true`
+         */
+        clickable?: boolean;
+        /**
+         * Allow double click to toggle fullscreen state, default: `true`
+         */
+        dblclickable?: boolean;
+        /**
+         * Add your own shortcuts
+         */
+        shortcuts?: Array<ShortcutItem>;
+    }
+
+    interface PosterImageProps {
+        /**
+         * A URL indicating a poster frame to show until the user plays or
+         * seeks. If this attribute isn't specified, nothing is displayed
+         * until the first frame is available; then the first frame is shown
+         * as the poster frame.
+         */
+        poster: string;
+    }
+
+    interface ControlBarProps {
+        /**
+         * Hide the control bar automatically after the player is inactive,
+         * default: `true`
+         */
+        autoHide?: boolean;
+        /**
+         * The waiting time for auto hide after player is inactive (in milliseconds),
+         * default: `3000`
+         */
+        autoHideTime?: number;
+        /**
+         * Do not render default controls, only use custom ones provided as children
+         * of `<ControlBar>`
+         */
+        disableDefaultControls?: boolean;
+        /**
+         * Do not render the control bar if set it to true, default: `false`
+         */
+        disableCompletely?: boolean;
+    }
+
+    interface ReplayControlProps {
+        /**
+         * How many seconds to go forward, default: `10`
+         */
+        seconds?: 5 | 10 | 30;
+    }
+
+    interface BigPlayButtonProps {
+        /**
+         * Determines the position of the big play button.
+         */
+        position?: 'center' | 'left' | 'left-top' | 'left-bottom' | 'right' | 'right-top' | 'right-bottom';
+    }
+
+    interface ForwardControlProps {
+        /**
+         * How many seconds to go forward, default: `10`
+         */
+        seconds?: 5 | 10 | 30;
+    }
+
+    interface VolumeMenuButtonProps {
+        /**
+         * The direction where Volume Bar popup, default: `false`
+         */
+        vertical?: boolean;
+    }
+
+    interface PlaybackRateMenuButtonProps {
+        /**
+         * The direction where Volume Bar popup, default:
+         * `[2, 1.5, 1.25, 1, 0.5, 0.25]`
+         */
+        rates?: Array<number>;
```

Comment:
Same syntax for arrays as above should be used here

```suggestion
        rates?: number[];
```